src/static/js/event_new.js

```diff
@@ -0,0 +1,183 @@
+/*global $, jQuery, _, additional, asm, common, html */
+
+$(function() {
+
+    "use strict";
+
+    const event_new = {
+
+        render: function(){
+            return [
+                '<div id="dialog-similar" style="display: none" title="' + html.title(_("Similar Event")) + '">',
+                '<p><span class="ui-icon ui-icon-alert"></span>',
+                _("This event is very similar to another event on file, carry on creating this record?"),
+                '<br /><br />',
+                '<span class="similar-event"></span>',
+                '</p>',
+                '</div>',
```

Comment:
Not sure that similar checks are really needed for events? I would have thought it unlikely that people would create duplicates.

Reply:
@bobintetley we also think it's not likely for people to create dups and so haven't implemented it yet but just copied this part for consistency with the rest of the system, but we figured that it may be worth checking for dupes (by dates+location for instance) just to make it more friendly. in SOS Pets, there are weekly events, and if 2 people create the same event, it might cause a fuss